build-farm/make-adopt-build-farm.sh~

```diff
@@ -0,0 +1,246 @@
+#!/bin/bash
```

Comment:
file added by mistake?

Reply:
Yep - my shell's been tab expanding to backup files as a priority recently ... I'll remove